arm-only.mk

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
I don't see this file being used as at all, and it shouldn't be needed. Can we remove it?